file.go

```diff
@@ -0,0 +1,79 @@
+package pluginapi
+
+import (
+	"bytes"
+	"io"
+	"io/ioutil"
+
+	"github.com/mattermost/mattermost-server/v5/model"
+	"github.com/mattermost/mattermost-server/v5/plugin"
+)
+
+// FileService provides features to deal with post attachments.
+type FileService struct {
+	api plugin.API
+}
+
+// Get gets content of a file by id.
+//
+// @tag File minimum server version: 5.8
```

Comment:
We can probably eliminate all the `@tag` stuff, now that the grouping is obvious given the service.